src/Console/ConfigurationResolver.php

```diff
@@ -737,7 +737,7 @@ function ($path) {
             if (is_file($path)) {
                 $pathsByType['file'][] = $path;
             } else {
-                $pathsByType['dir'][] = $path.DIRECTORY_SEPARATOR;
+                $pathsByType['dir'][] = substr($path, -1, 1) !== DIRECTORY_SEPARATOR ? $path : substr($path, 0, strlen($path) - 1);
```

Comment:
This will fail if the path is `/`: the value will be an empty string, resulting in an exception being thrown:
```
The "" directory does not exist.
```